oracle/cmd/main.go

```diff
@@ -242,6 +242,22 @@ var (
 		EnvVars: []string{"MEV_COMMIT_GAS_FEE_CAP"},
 		Value:   "2000000000", // 2 gWEI
 	})
+	optionRelayUrls = altsrc.NewStringSliceFlag(&cli.StringSliceFlag{
+		Name:    "relay-urls",
+		Usage:   "URLs for relay",
+		EnvVars: []string{"MEV_ORACLE_RELAY_URLS"},
+		Value: cli.NewStringSlice(
+			"https://boost-relay.flashbots.net/relay/v1/data/bidtraces/proposer_payload_delivered",
+			"https://bloxroute.max-profit.blxrbdn.com/relay/v1/data/bidtraces/proposer_payload_delivered",
```

Comment:
Need to have a clean way to decision if these should be holesky v.s mainnet values.